### PR TITLE
Require php 8

### PR DESCRIPTION
**Pull request by MortalFlesh**

### CODE REVIEW

File: ecs.php [hunk 1]
```diff
@@ -13,7 +15,11 @@
 
     $parameters->set(
         Option::SKIP,
-        ['SlevomatCodingStandard\Sniffs\Exceptions\ReferenceThrowableOnlySniff.ReferencedGeneralException' => ['tests/Exception/*.php']]
+        [
+            'SlevomatCodingStandard\Sniffs\Exceptions\ReferenceThrowableOnlySniff.ReferencedGeneralException' => ['tests/Exception/*.php'],
+            OperatorSpacingSniff::class => null,
+            BinaryOperatorSpacesFixer::class => null,
```
Comment (OndraM) [inline ecs.php:21]: What about adding [this](https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/5495#issuecomment-777187487) directly to lmc/coding-standard?

File: ecs.php [hunk 2]
```diff
@@ -13,7 +15,11 @@
 
     $parameters->set(
         Option::SKIP,
-        ['SlevomatCodingStandard\Sniffs\Exceptions\ReferenceThrowableOnlySniff.ReferencedGeneralException' => ['tests/Exception/*.php']]
+        [
+            'SlevomatCodingStandard\Sniffs\Exceptions\ReferenceThrowableOnlySniff.ReferencedGeneralException' => ['tests/Exception/*.php'],
+            OperatorSpacingSniff::class => null,
```
Comment (OndraM) [inline ecs.php:20]: I guess this is about spaces around union types; is there a related issue in php codesfniffer?

File: ecs.php (hunk 1 above)
Comment (MortalFlesh) [reply ecs.php:21]: AFAIK it should be handled properly in phpcs 3.6.0

File: ecs.php (hunk 2 above)
Comment (MortalFlesh) [reply ecs.php:20]: it doesn't support php 8 yet

**Review by legendik — APPROVED** [review]

**Pull request merged**